test/integration/functional_test.go

```diff
@@ -59,6 +62,7 @@ func TestFunctional(t *testing.T) {
 			name      string
 			validator validateFunc
 		}{
+			{"CopySyncFile", validateCopySynFile},      // Set file for the file sync test case
```

Comment:
Naming wise not 100% confident it is correct but just to follow the rest of the test case. Would prefer to name it setupCopySyncFile(..) to make it more readable.

Reply:
I like your idea as well - let's do that. You could even go shorter with the name:`setupFileSync`